### PR TITLE
Add circuit breaker check

### PR DESCRIPTION
**Pull request by madAndroid**
Triggered circuit breakers have caused issues for us recently, this should help catch those

### CODE REVIEW

**Comment by eheydrick** [comment]
@madAndroid Thanks! I've added rubocop exclusions that should allow the tests to pass. Can you add the new plugin to the README and CHANGELOG.

**Comment by madAndroid** [comment]
Awesome, cheers @eheydrick - I was wondering how I'd get around that particular check :) 
I've updated the CHANGELOG and README, as requested.

File: bin/check-es-circuit-breakers.rb [hunk 1]
```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env ruby
+#
+#  check-es-node-status
```
Comment (eheydrick) [inline bin/check-es-circuit-breakers.rb]: should be check-es-circuit-breakers

File: bin/check-es-circuit-breakers.rb [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: rest-client
+#
+# USAGE:
+#   check-es-node-status --help
```
Comment (eheydrick) [inline bin/check-es-circuit-breakers.rb]: should have usage for check-es-circuit-breakers

File: bin/check-es-circuit-breakers.rb [hunk 3]
```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env ruby
+#
+#  check-es-node-status
+#
+# DESCRIPTION:
+#   This plugin checks the ElasticSearch node status, using its API.
+#   Works with ES 0.9x and ES 1.x
```
Comment (eheydrick) [inline bin/check-es-circuit-breakers.rb]: update for this plugin

File: bin/check-es-circuit-breakers.rb [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+#   Copyright 2012 Sonian, Inc <chefs@sonian.net>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'rest-client'
+require 'json'
+require 'base64'
+require 'pp'
```
Comment (eheydrick) [inline bin/check-es-circuit-breakers.rb]: pp doesn't appear to be used.

**Comment by eheydrick** [comment]
@madAndroid Left a few other comments, will be good to go after those are resolved.

**Pull request merged**